### ISSUE TITLE
[Whitelist] Pro Evolution Soccer 2018 Lite (592600)

### ISSUE THREAD

**Issue by sgripon** [ValveSoftware/Proton#305]
Played several games, no problems.

Proton 3.7

System:    Kernel: 4.15.0-32-generic x86_64 bits: 64 Desktop: Gnome 3.28.2
           Distro: Ubuntu 18.04.1 LTS
Machine:   Device: desktop System: Micro-Star product: MS-7B49 v: 1.0 serial: N/A
           Mobo: Micro-Star model: Z370 PC PRO (MS-7B49) v: 1.0 serial: N/A
           UEFI: American Megatrends v: 1.20 date: 12/21/2017
Battery    hidpp__0: charge: N/A condition: NA/NA Wh
CPU:       6 core Intel Core i5-8600K (-MCP-) cache: 9216 KB
           clock speeds: max: 4300 MHz 1: 3437 MHz 2: 3937 MHz 3: 3325 MHz 4: 3851 MHz 5: 3651 MHz 6: 4219 MHz
Graphics:  Card: NVIDIA GP106 [GeForce GTX 1060 6GB]
           Display Server: x11 (X.Org 1.19.6 ) drivers: nvidia (unloaded: modesetting,fbdev,vesa,nouveau)
           Resolution: 1280x1024@60.02hz, 1920x1080@74.97hz
           OpenGL: renderer: GeForce GTX 1060 6GB/PCIe/SSE2 version: 4.6.0 NVIDIA 390.48
Audio:     Card-1 NVIDIA GP106 High Definition Audio Controller driver: snd_hda_intel
           Card-2 Intel 200 Series PCH HD Audio driver: snd_hda_intel
           Sound: Advanced Linux Sound Architecture v: k4.15.0-32-generic
Network:   Card: Intel Ethernet Connection (2) I219-V driver: e1000e
           IF: enp0s31f6 state: up speed: 100 Mbps duplex: full mac: 30:9c:23:8c:77:3a
Drives:    HDD Total Size: 2120.4GB (39.4% used)
           ID-1: /dev/sda model: Corsair_Force_LE size: 120.0GB
           ID-2: /dev/sdb model: ST2000DM006 size: 2000.4GB
Partition: ID-1: / size: 110G used: 17G (17%) fs: ext4 dev: /dev/sda2
RAID:      No RAID devices: /proc/mdstat, md_mod kernel module present

**Comment by leillo1975** [comment]
Pro Evolution soccer has a configuration utility when you start the game (in Windows), where you can set the game in fullscreen. In linux, it's not shown at the start and you only play in a Window

**Comment by daxcore** [comment]
Yeah its important to start steam in not-native mode. To get fullscreen, resolution and controller, I copied the *.dat file from windows. But performance isnt really well, middle settings does lag on gtx1080.

**Comment by carzacc** [comment]
It doesn't seem to work with AMD hardware and Mesa drivers, at least on my machine.
The same is true for the paid version.

**Comment by GitoMat** [comment]
@carzacc Make sure that you have installed the Vulkan drivers:
`sudo apt install mesa-vulkan-drivers mesa-vulkan-drivers:i386`
I missed that first, now it works on Ubuntu 18.04.1 with AMD Mesa.

**Comment by carzacc** [comment]
I already had mesa-vulkan-drivers installed (the 32bit one is mesa-vulkan-drivers.i686 since I'm on Fedora but it's the same), now I'll try with Ubuntu to see if it's a distribution-related issue.

**Comment by GitoMat** [comment]
I forgot to mention this was with PES 2019 full version, but I imagine they shouldn't differ too much in requirements.

**Comment by alkazar** [comment]
This game actually stopped working for me after a steam client or proton update. It no longer launches and just crashes immediately before displaying anything to the screen. I am using latest SteamOS beta, Steam client beta and Steam Play beta with Nvidia 396.54. Is anyone still able to play this game?
